pkg/helper/helpers.go

```diff
@@ -378,6 +378,22 @@ func AppliedManifestworkHubHashFilter(hubHash string) factory.EventFilterFunc {
 	}
 }
 
+// AppliedManifestWorkEvictionFilter filter the appliedmanifestwork belonging to this hub or this work agent
+func AppliedManifestWorkEvictionFilter(hubHash, agentID string) factory.EventFilterFunc {
```

Comment:
We need some explanation of this filter, why evictionfilter needs to consider hub hash and agent id. Or we name it as AppliedManifestWorkFilterByHubHashOrAgentID